### PR TITLE
[zh-cn]: add the translation of JavaScript code example style guideline

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#11063

### CODE REVIEW

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,876 @@
...
+### 多行注释
+
+简短的注释通常更好，所以尝试将他们保持在 60–80 个字符的单行中。如果这不可能，请在每行的开头使用 `//`：
+
+```js example-good
+// This is an example of a multi-line comment.
+// The imaginary function that follows has some unusual
+// limitations that I want to call out.
+// Limitation 1
+// Limitation 2
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:175]:
```suggestion
// 这是一个多行注释的示例。
// 假设下面的函数有一些我想指出的不寻常的局限性。
// 局限 1
// 局限 2
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 2]
```diff
@@ -0,0 +1,876 @@
...
+```
+
+请不要使用 `/* … */`:
+
+```js example-bad
+/* This is an example of a multi-line comment.
+  The imaginary function that follows has some unusual
+  limitations that I want to call out.
+  Limitation 1
+  Limitation 2 */
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:185]:
```suggestion
/* 这是一个多行注释的示例。
  假设下面的函数有一些我想指出的不寻常的局限性。
  局限 1
  局限 2 */
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 3]
```diff
@@ -0,0 +1,876 @@
...
+
+而不要这样创建数组：
+
+```js example-bad
+const visitedCities = new Array(length);
+```
+
+### 添加元素
+
+向数组添加元素时，请使用 [`push()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array/push) 而非直接赋值。考虑如下数组：
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:46]:
```suggestion
向数组添加元素时，请使用 [`push()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array/push) 而非直接赋值。考虑有如下数组：
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 4]
```diff
@@ -0,0 +1,876 @@
...
+编写异步代码可以提高性能，应尽可能使用。特别是，你可以使用：
+
+- [Promises](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+- [`async`](/zh-CN/docs/Web/JavaScript/Reference/Statements/async_function)/[`await`](/zh-CN/docs/Web/JavaScript/Reference/Operators/await)
+
+当两种语法都可行时，我们倾向于使用更简单的 `async`/`await` 语法。不幸的是，你不能在顶层使用 `await`，除非你在 ECMAScript 模块中。但 Node.js 使用 CommonJS 模块而不是 ES 模块。因此，如果你打算在所有地方使用该示例，请避免顶层的 `await`。
+
+## 注释
+
+注释对于编写好的代码示例非常重要。它们清楚地阐明了代码的意图，并帮助开发者理解它。要特别注意它们。
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:75]:
```suggestion
注释对于编写良好的代码示例非常重要。它们清楚地阐明了代码的意图，并帮助开发者理解它。因此要特别注意它们。
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 5]
```diff
@@ -0,0 +1,876 @@
...
+- 当函数的名称清楚地描述了它们的作用时，不需要再注释。如：
+
+  ```js example-good
+  closeConnection();
+  ```
+
+  不要写像下面这样的注释：
+
+  ```js example-bad
+  closeConnection(); // Closing the connection
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:109]:
```suggestion
  closeConnection(); // 关闭连接
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 6]
```diff
@@ -0,0 +1,876 @@
...
+    // 创建一个文档并向其添加一个新的段落
+    const para = document.createElement("p");
+    para.textContent = `总价是 ${total}`;
+    document.body.appendChild(para);
+  }
+  ```
+
+### 打印的输出
+
+- 对于预期在生产环境中运行的代码，你很少会需要对打印代码进行注释。在代码示例中，我们经常使用 `console.log()`、`console.error()` 或类似的函数来输出重要的值。为了帮助读者在不运行代码的情况下理解会发生什么，你可以在打印函数*之后*注释输出的内容。比如：
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:140]:
```suggestion
- 对于预期在生产环境中运行的代码，你很少会需要对打印语句进行注释。在代码示例中，我们经常使用 `console.log()`、`console.error()` 或类似的函数来输出重要的值。为了帮助读者在不运行代码的情况下理解会发生什么，你可以在打印函数*之后*注释输出的内容。比如：
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 7]
```diff
@@ -0,0 +1,874 @@
...
+  closeConnection(); // 关闭连接
+  ```
+
+### 使用单行注释
+
+单行注释用 `//` 标记，与包含在 `/* … */` 之间的块注释相反。
+
+通常情况下，使用单行注释来注释代码。编写者必须用 `//` 标记每一行的注释，这样就更容易直观地注意到注释掉的代码。此外，这种约定允许在调试时使用 `/* … */` 注释掉部分代码。
+
+- 在斜杠和空格之间留一个空格。注释以大写字母开头，就像句子一样，但不要在结尾加句号。
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:118]:
```suggestion
- 在斜杠和空格之间留一个空格。（英文的）注释以大写字母开头，就像句子一样，但不要在结尾加句号。
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 8]
```diff
@@ -0,0 +1,874 @@
...
+  function exampleFunc(fruitBasket) {
+    console.log(fruitBasket); // ['banana', 'mango', 'orange']
+  }
+  ```
+
+  请不要这样写：
+
+  ```js example-bad
+  function exampleFunc(fruitBasket) {
+    // Logs: ['banana', 'mango', 'orange']
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:152]:
```suggestion
    // 输出：['banana', 'mango', 'orange']
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 9]
```diff
@@ -0,0 +1,874 @@
...
+简短的注释通常更好，所以尝试将他们保持在 60–80 个字符的单行中。如果这不可能，请在每行的开头使用 `//`：
+
+```js example-good
+// 这是一个多行注释的示例。
+// 假设下面的函数有一些我想指出的不寻常的局限性。
+// 局限 1
+// 局限 2
+```
+
+请不要使用 `/* … */`:
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:177]:
```suggestion
请不要使用 `/* … */`：
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 10]
```diff
@@ -0,0 +1,874 @@
...
+
+  不要这样：
+
+  ```js example-bad
+  const x = () => {
+    // …
+  };
+  ```
+
+- 使用箭头函数时，尽可能使用[隐式返回](/zh-CN/docs/Web/JavaScript/Reference/Functions/Arrow_functions#函数体)（也称为*表达式体*）：
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:301]:
```suggestion
- 使用箭头函数时，尽可能使用[隐式返回](/zh-CN/docs/Web/JavaScript/Reference/Functions/Arrow_functions#函数体)（也称为*表达式主体*）：
```

**Review by yin1999 — COMMENTED** [review]
reviewed to line 315. The first comment is used for self note, please ignore this for now

**Review by yin1999 — COMMENTED** [review]
finished review. Could you please translate the strings in examples where appropriate

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 11]
```diff
@@ -0,0 +1,874 @@
+---
+title: JavaScript 代码示例编写指南
+slug: MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript
+---
+
+{{MDNSidebar}}
+
+以下指南涵盖了如何为 MDN Web Docs 编写 JavaScript 示例代码。本文列出了编写简洁示例的规则，以期这些示例将被尽可能多的人理解。
+
+## JavaScript 代码示例的一般准则
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:10]: Could we align with other parts: https://github.com/mdn/translated-content/pull/17048/files#diff-5c60d8166e0b093abe61ed358df027b28d8de216ac01bbb8bbcf5b8b61c9d5cfR24-R30

```suggestion
## JavaScript 代码示例常规指南
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 12]
```diff
@@ -0,0 +1,873 @@
...
+  const dogs = ["Rex", "Lassie"];
+  dogs.forEach((dog) => {
+    console.log(dog);
+  });
+  ```
+
+  请不要使用 `for (;;)`——你不仅需要添加额外的索引 `i`，你还必须跟踪数组的长度。这对于初学者来说很容易出错。
+
+  ```js example-bad
+  const dogs = ["Rex", "Lassie"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:344]:
```suggestion
  const dogs = ["旺财", "来福"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 13]
```diff
@@ -0,0 +1,873 @@
...
+
+  for (let i = 0; i < 4; i++) {
+    result += arr[i];
+  }
+  ```
+
+  下面的示例没有遵循推荐的初始化准则（它隐式创建了全局变量，并且在严格模式下会失败）：
+
+  ```js example-bad
+  const cats = ["Athena", "Luna"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:366]:
```suggestion
  const cats = ["汤姆", "胖橘"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 14]
```diff
@@ -0,0 +1,873 @@
...
+  for (i of cats) {
+    console.log(i);
+  }
+  ```
+
+- 但是需要同时范围值和索引时，你可以使用 `.forEach()` 而不是 `for (;;)`。例如：
+
+  ```js example-good
+  const gerbils = ["Zoé", "Chloé"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:375]:
```suggestion
  const gerbils = ["舒克", "贝塔"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 15]
```diff
@@ -0,0 +1,873 @@
...
+  gerbils.forEach((gerbil, i) => {
+    console.log(`Gerbil #${i}: ${gerbil}`);
+  });
+  ```
+
+  请不要这样写：
+
+  ```js example-bad
+  const gerbils = ["Zoé", "Chloé"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:384]:
```suggestion
  const gerbils = ["舒克", "贝塔"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 16]
```diff
@@ -0,0 +1,873 @@
...
+}
+```
+
+### 将大括号与控制流语句和循环一起使用
+
+虽然像 `if`、`for` 和 `while` 的控制流语句不需要在内容中只有一个语句时使用大括号，但你总是应该使用大括号。比如：
+
+```js example-good
+for (const car of storedCars) {
+  car.paint("red");
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:430]:
```suggestion
  car.paint("红色");
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 17]
```diff
@@ -0,0 +1,873 @@
...
+}
+```
+
+不要这样写：
+
+```js example-bad
+for (const car of storedCars) car.paint("red");
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:437]:
```suggestion
for (const car of storedCars) car.paint("红色");
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 18]
```diff
@@ -0,0 +1,873 @@
...
+  ```js example-good
+  class Person {
+    constructor(name, age, pronouns) {
+      this.name = name;
+      this.age = age;
+      this.pronouns = pronouns;
+    }
+
+    greeting() {
+      console.log(`Hi! I'm ${this.name}`);
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:566]:
```suggestion
      console.log(`你好！我是 ${this.name}`);
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 19]
```diff
@@ -0,0 +1,873 @@
...
+### 循环初始化
+
+当需要[循环](/zh-CN/docs/Learn/JavaScript/Building_blocks/Looping_code)时，从 [`for(;;)`](/zh-CN/docs/Web/JavaScript/Reference/Statements/for)、[`for...of`](/zh-CN/docs/Web/JavaScript/Reference/Statements/for...of)、[`while`](/zh-CN/docs/Web/JavaScript/Reference/Statements/while) 等中选择合适的。
+
+- 在遍历集合中的所有元素时，避免使用传统的 `for (;;)` 循环；优先使用 `for...of` 或 `forEach()`。请注意，如果你使用的是一个不是 `Array` 的集合，你必须检查 `for...of` 是否真的被支持（它需要变量是可迭代的），或者 `forEach()` 方法是否存在。
+
+  使用 `for...of`：
+
+  ```js example-good
+  const dogs = ["Rex", "Lassie"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:326]:
```suggestion
  const dogs = ["旺财", "来福"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 20]
```diff
@@ -0,0 +1,873 @@
...
+  for (const dog of dogs) {
+    console.log(dog);
+  }
+  ```
+
+  或 `forEach()`：
+
+  ```js example-good
+  const dogs = ["Rex", "Lassie"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:335]:
```suggestion
  const dogs = ["旺财", "来福"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 21]
```diff
@@ -0,0 +1,873 @@
...
+  const dogs = ["旺财", "来福"];
+  for (let i = 0; i < dogs.length; i++) {
+    console.log(dogs[i]);
+  }
+  ```
+
+- 确保你正确地定义了初始化器，对 `for...of` 使用 `const` 关键字，或对其他循环使用 `let`。注意不要遗漏。以下是正确的示例：
+
+  ```js example-good
+  const cats = ["Athena", "Luna"];
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:353]:
```suggestion
  const cats = ["汤姆", "胖橘"];
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 22]
```diff
@@ -0,0 +1,873 @@
...
+    console.log(i);
+  }
+  ```
+
+- 但是需要同时范围值和索引时，你可以使用 `.forEach()` 而不是 `for (;;)`。例如：
+
+  ```js example-good
+  const gerbils = ["舒克", "贝塔"];
+  gerbils.forEach((gerbil, i) => {
+    console.log(`Gerbil #${i}: ${gerbil}`);
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:377]:
```suggestion
    console.log(`沙鼠 ${i}号：${gerbil}`);
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 23]
```diff
@@ -0,0 +1,873 @@
...
+  });
+  ```
+
+  请不要这样写：
+
+  ```js example-bad
+  const gerbils = ["舒克", "贝塔"];
+  for (let i = 0; i < gerbils.length; i++) {
+    console.log(`Gerbil #${i}: ${gerbils[i]}`);
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:386]:
```suggestion
    console.log(`沙鼠 ${i}号：${gerbil}`);
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 24]
```diff
@@ -0,0 +1,873 @@
...
+  ```js example-good
+  class Person {
+    constructor(name, age, pronouns) {
+      this.name = name;
+      this.age = age;
+      this.pronouns = pronouns;
+    }
+
+    greeting() {
+      console.log(`Hi! I'm ${this.name}`);
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:566]:
```suggestion
      console.log(`你好！我是 ${this.name}`);
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 25]
```diff
@@ -0,0 +1,873 @@
...
+
+### 模版字面量
+
+要将值插入字符串，请使用[模板字面量](/zh-CN/docs/Web/JavaScript/Reference/Template_literals)。
+
+- 这里是一个推荐的使用模板字面量的例子。它们的使用可以防止大量的空格错误。
+
+  ```js example-good
+  const name = "Shilpa";
+  console.log(`Hi! I'm ${name}!`);
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:687]:
```suggestion
  console.log(`你好！我是 ${name}!`);
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 26]
```diff
@@ -0,0 +1,873 @@
...
+  ```
+
+  请不要像这样串联字符串：
+
+  ```js example-bad
+  const name = "Shilpa";
+  console.log("Hi! I'm" + name + "!"); // Hi! I'mShilpa!
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:694]:
```suggestion
  console.log("你好！我是 " + name + "！"); // 你好！我是 Shilpa！
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 27]
```diff
@@ -0,0 +1,873 @@
...
+  const name = "Shilpa";
+  console.log(name);
+  ```
+
+- 如果你会改变变量的值，请使用 `let`，如下所示：
+
+  ```js example-good
+  let age = 40;
+  age++;
+  console.log("Happy birthday!");
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:743]:
```suggestion
  console.log("生日快乐！");
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 28]
```diff
@@ -0,0 +1,873 @@
...
+  let name = "Shilpa";
+  console.log(name);
+  ```
+
+- 下面的示例使用 `const` 声明了一个变量，该变量会被重新赋值。重新赋值会抛出一个错误。
+
+  ```js example-bad
+  const age = 40;
+  age++;
+  console.log("Happy birthday!");
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:758]:
```suggestion
  console.log("生日快乐！");
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 29]
```diff
@@ -0,0 +1,873 @@
...
+  var age = 40;
+  var name = "Shilpa";
+  ```
+
+- 一行声明一个变量，如下所示：
+
+  ```js example-good
+  let var1;
+  let var2;
+  let var3 = "Apapou";
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:773]:
```suggestion
  let var3 = "张三";
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 30]
```diff
@@ -0,0 +1,873 @@
...
+  let var4 = var3;
+  ```
+
+  不要在一行中通过逗号或链式声明来声明多个变量。避免像这样声明变量：
+
+  ```js-nolint example-bad
+  let var1, var2;
+  let var3 = var4 = "Apapou"; // var4 被隐式创建为全局变量；这在严格模式下会失败
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:781]:
```suggestion
  let var3 = var4 = "张三"; // var4 被隐式创建为全局变量；这在严格模式下会失败
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 31]
```diff
@@ -0,0 +1,873 @@
...
+- 在 Web 音频 API 中使用 `AudioWorklet` 来替代 `ScriptProcessorNode`。
+
+### 使用安全可靠的 API
+
+- 请不要使用 {{DOMxRef("Element.innerHTML")}} 将纯文本内容插入到元素中；而是使用 {{DOMxRef("Node.textContent")}} 替代。`innerHTML` 属性可能会导致安全问题，如果开发人员不控制参数。作为编写者，我们越避免使用它，开发者复制和粘贴我们的代码时产生的安全漏洞就越少。
+
+  下面的示例演示了 `textContent` 的用法。
+
+  ```js example-good
+  const text = "Hello to all you good people";
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:851]:
```suggestion
  const text = "父老乡亲们大家好";
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 32]
```diff
@@ -0,0 +1,873 @@
...
+  const para = document.createElement("p");
+  para.textContent = text;
+  ```
+
+  不要使用 `innerHTML` 将*纯文本*插入到 DOM 节点中。
+
+  ```js example-bad
+  const text = "Hello to all you good people";
```
Comment (JasonLamv-t) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:859]:
```suggestion
  const text = "父老乡亲们大家好";
```

**Comment by JasonLamv-t** [comment]
> finished review. Could you please translate the strings in examples where appropriate

I have translated some strings in the code examples.

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 33]
```diff
@@ -0,0 +1,873 @@
...
+    console.log(i);
+  }
+  ```
+
+- 但是需要同时范围值和索引时，你可以使用 `.forEach()` 而不是 `for (;;)`。例如：
+
+  ```js example-good
+  const gerbils = ["舒克", "贝塔"];
+  gerbils.forEach((gerbil, i) => {
+    console.log(`沙鼠 ${i}号：${gerbil}`);
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:377]:
```suggestion
    console.log(`沙鼠 ${i} 号：${gerbil}`);
```

File: files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md [hunk 34]
```diff
@@ -0,0 +1,873 @@
...
+  });
+  ```
+
+  请不要这样写：
+
+  ```js example-bad
+  const gerbils = ["舒克", "贝塔"];
+  for (let i = 0; i < gerbils.length; i++) {
+    console.log(`沙鼠 ${i}号：${gerbil}`);
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/writing_style_guide/code_style_guide/javascript/index.md:386]:
```suggestion
    console.log(`沙鼠 ${i} 号：${gerbil}`);
```

**Review by yin1999 — APPROVED** [review]
Thanks @JasonLamv-t :heart:

**Pull request merged**